### ISSUE TITLE
gclip value

### ISSUE THREAD

**Issue by harshraj22** [sanghyun-son/EDSR-PyTorch#312]
https://github.com/sanghyun-son/EDSR-PyTorch/blob/9d3bb0ec620ea2ac1b5e5e7a32b0133fbba66fd2/src/trainer.py#L51

Hello.
What value did you use for gradient clipping while training ?
`demo.sh` has no example to train the edsr model.
